iidm/iidm-api/src/main/java/com/powsybl/iidm/network/NetworkListener.java

```diff
@@ -17,4 +17,8 @@
 
     void onUpdate(Identifiable identifiable, String attribute, Object oldValue, Object newValue);
 
+    default void onUpdate(Identifiable identifiable, String attribute, int variantIndex, Object oldValue,
```

Comment:
Variant index is an implementation detail, we should notify using variant name instead.

Reply:
Agree! @tadam50 could you change this?